README.md

```diff
@@ -527,38 +570,137 @@ respect cancellation, the shutdown may never complete.
+#### Use
+
+The wheel can now be installed into any virtual environment.
+
+For example,
+[create a virtual environment](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments)
+somewhere and then run the following inside the virtual environment:
+
+```bash
+pip install /path/to/cloned/sdk-python/dist/*.whl
+```
+
+Create this Python file at `example.py`:
+
+```python
+import asyncio
+from temporalio import workflow, activity
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+@workflow.defn
+class SayHello:
+    @workflow.run
+    async def run(self, name: str) -> str:
+        return f"Hello, {name}!"
+
+async def main():
+    client = await Client.connect("http://localhost:7233")
+    async with Worker(client, task_queue="my-task-queue", workflows=[SayHello]):
+        result = await client.execute_workflow(SayHello.run, "Temporal",
+            id="my-workflow-id", task_queue="my-task-queue")
+        print(f"Result: {result}")
 
-- Install the system dependencies:
+if __name__ == "__main__":
+    asyncio.run(main())
+```
 
-  - Python >=3.7
-  - [pipx](https://github.com/pypa/pipx#install-pipx) (only needed for installing the two dependencies below)
-  - [poetry](https://github.com/python-poetry/poetry) `pipx install poetry`
-  - [poe](https://github.com/nat-n/poethepoet) `pipx install poethepoet`
+Assuming there is a [local Temporal server](https://docs.temporal.io/docs/server/quick-install/) running, executing the
+file with `python` (or `python3` if necessary) will give:
 
-- Use a local virtual env environment (helps IDEs and Windows):
+    Result: Hello, Temporal!
 
-  ```bash
-  poetry config virtualenvs.in-project true
-  ```
+### Local SDK development environment
 
-- Install the package dependencies (requires Rust):
+For local development, it is often quicker to use debug builds and a local virtual environment.
 
-  ```bash
-  poetry install
-  ```
+While not required, it often helps IDEs if we put the virtual environment `.venv` directory in the project itself. This
+can be configured system-wide via:
 
-- Build the project (requires Rust):
+```bash
+poetry config virtualenvs.in-project true
+```
 
-  ```bash
-  poe build-develop
-  ```
+Now perform the same steps as the "Prepare" section above by installing the prerequisites, cloning the project,
+installing dependencies, and generating the protobuf code:
 
-- Run the tests (requires Go):
+```bash
+git clone --recursive https://github.com/temporalio/sdk-python.git
+cd sdk-python
+poetry install --no-root
+poe gen-protos
+```
 
-  ```bash
-  poe test
-  ```
+Now compile the Rust extension in develop mode which is quicker than release mode:
+
+```bash
+poe build-develop
+```
+
+That step can be repeated for any Rust changes made.
+
+The environment is now ready to develop in.
+
+#### Testing
+
+Tests currently require [Go](https://go.dev/) to be installed since they use an embedded Temporal server as a library.
```

Comment:
Are you going to get rid of this soon?

Reply:
There are two things in Go right now: Temporalite wrapper and "kitchen-sink" workflow/worker. I am waiting on TLS support and distributed Temporalite binaries for the former. For the latter, probably at the same time I get rid of the former, I will rewrite the workflow in Python and get rid of the Go worker.